q2_phylogeny/plugin_setup.py

```diff
@@ -409,6 +410,36 @@
                  "are not tip identifiers in tree.")
 )
 
+plugin.methods.register_function(
+    function=q2_phylogeny.robinson_foulds,
+    inputs={'trees': List[Phylogeny[Rooted | Unrooted]]},
+    parameters={
+        'labels': List[Str],
+        'missing_tips': Str % Choices('error', 'intersect-all')
+    },
+    outputs=[('distance_matrix', DistanceMatrix)],
+    input_descriptions={
+        'trees': 'Phylogenetic trees to compare with Robinson-Foulds. Rooting'
+                 ' information and branch lengths are ignored by this metric.'
+    },
+    parameter_descriptions={
+        'labels': 'Labels to use for the tree names in the distance matrix.'
+                  ' If ommited, labels will be "tree_n" where "n" ranges from'
+                  ' 1..N. The number of labels must match the number of'
+                  ' trees.',
+        'missing_tips': 'How to handle tips that are not shared between trees.'
```

Comment:
The only suggestion I have is on this text. The following is clearer to me:

"error" will raise an error if the set of tips is not identical between all input trees. "intersect-all" will remove tips that are not shared between all trees before computing distances between trees.